backends/intel_gpu/toolkit/actual_expect.diff

```diff
@@ -0,0 +1,29 @@
+diff --git a/python/paddle/fluid/tests/unittests/op_test.py b/python/paddle/fluid/tests/unittests/op_test.py
+index d672dccec9..95413e733c 100644
```

Comment:
please create an independent file of “op_test.py” for intel gpu, if it have many differences btw Paddle's op_test.py file instead of maintain an diff file here.

Reply:
file removed